tests/components/rfxtrx/test_switch.py

```diff
@@ -151,3 +152,24 @@ async def test_discover_rfy_sun_switch(hass, rfxtrx_automatic):
     state = hass.states.get("switch.rfy_030101_1")
     assert state
     assert state.state == "on"
+
+
+async def test_unknown_event_code(hass, rfxtrx):
+    """Test with 3 switches."""
+    assert await async_setup_component(
+        hass,
+        "rfxtrx",
+        {
+            "rfxtrx": {
+                "device": "abcd",
+                "devices": {"1234567890": {}},
+            }
+        },
+    )
+    await hass.async_block_till_done()
+
+    conf_entries = hass.config_entries.async_entries(DOMAIN)
+    assert len(conf_entries) == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6090606)